### ISSUE TITLE
[Investigate] NGINX binary compiled for debugging

### ISSUE THREAD

**Issue by mvalliath** [paketo-buildpacks/nginx#377]
We have received a user request to enable debugging in the NGINX buildpack. This would require recompiling NGINX to include this debugging functionality.

End user goal: this user would like to enable log debugging for their NGINX app 

Desired outcomes:
* Level of complexity/effort 
* Whether there any drawbacks for replacing the current version of NGINX with a debugging-enabled version NGINX.

**Comment by sophiewigmore** [comment]
Also to consider: performance implications of this setting. There may be a reason that we don't have it set in the first place.

**Comment by robdimsdale** [comment]
Awesome thanks @TisVictress. Also did you notice any performance difference with debug logging on? I know we don't have a full performance test suite, but did your integration tests take noticeably longer, for example?

**Comment by TisVictress** [comment]
@robdimsdale Nope, there are no performance issues and the tests didn't take longer.

**Comment by TisVictress** [comment]
Created a new issue, #387, to implement these findings. Closing.

**Comment by arjun024** [comment]
Option 1 SGTM
